### DISCUSSION TITLE
新創公司開發語音分析演算法偵測憂鬱症前兆

### DISCUSSION THREAD

**Discussion by 110409538** [1101-GS4719-C/Python-Community-GS4719-C#660] [科技新知]
若這項技術可以成功,相信可以幫助更多人.但我覺得前提是本人和他們的家人願意接受測試.我覺得以後可以把憂鬱症的測試加入到每年的身體檢查的項目中.

新聞來源:https://technews.tw/2022/06/07/depressed-this-algorithm-can-tell-from-the-tone-of-your-voice/

**Comment by liangyinglly** [comment]
連結點不進去喔 我記得有人也分享過相關文章了

**Reply by 110409538** [reply]
連結已更新.對,我剛剛也看到了.<html>
<body>


:rofl:
--



</body>
</html>
